src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -141,24 +122,29 @@ const SubmissionDetails = _.flow(
           submitted && makeStatusLine(submittedIcon, `Submitted: ${submitted.length}`)
         ]),
         div({ style: { display: 'flex', flexWrap: 'wrap' } }, [
-          makeSection('Workflow Configuration',
+          makeSection('Workflow Configuration', [
             Utils.cond(
-              [methodAccessible, () => [h(Link,
-                { href: Nav.getLink('workflow', { namespace, name, workflowNamespace, workflowName }) },
-                [`${workflowNamespace}/${workflowName}`]
-              )]],
-              [methodAccessible === false,
-                () => [div({ style: { display: 'flex', alignItems: 'center' } }, [
-                  `${workflowNamespace}/${workflowName}`,
+              [methodAccessible, () => {
+                return h(Link, {
+                  href: Nav.getLink('workflow', { namespace, name, workflowNamespace, workflowName }),
+                  style: Style.noWrapEllipsis
+                }, [`${workflowNamespace}/${workflowName}`])
+              }],
+              [methodAccessible === false, () => {
+                return div({ style: { display: 'flex', alignItems: 'center' } }, [
+                  h(TooltipTrigger, {
+                    content: `${workflowNamespace}/${workflowName}` // TODO fix this width or wrap better
+                  }, [div({ style: Style.noWrapEllipsis }, [`${workflowNamespace}/${workflowName}`])]),
```

Comment:
I think you might just want TooltipCell here?
```suggestion
                                    h(TooltipCell, [`${workflowNamespace}/${workflowName}`]), // TODO fix this width or wrap better
```

Reply:
Happy to make the change. OOI, what's the difference between `TooltipCell` vs `TooltipTrigger`?

Reply:
Tooltip cell is basically a tooltip trigger wrapping around a div with no wrap ellipsis on it, with the same text in the tooltip and the div.

So it's exactly what you already were doing, just pre-made.